### DISCUSSION TITLE
Add husband-wife into relationship to family field when creating a volunteer family

### DISCUSSION THREAD

**Discussion by simply-emily** [CareTogether/CareTogetherCMS#182] [Ideas]
Like it says in the title. :-) We'd like a husband/wife option rather than just mom/dad.

**Comment by simply-emily** [comment]
Actually, please add: husband, wife, daughter, son

**Reply by LarsKemmann** [reply]
This has been added for ASF. @AGNMM18 - let me know if there are any additional relationship options (including the ones mentioned above) that you'd like to see for NMM, it's an easy config change.
